test/integration_test/specs/kubevirt-fedora-vm/portworx/sc-sharedv4svc-nolock-remote-only.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
```

Comment:
remove the "is-default-class" since it may conflict with other SCs?

Reply:
SC is not default now.